frontend/javascripts/oxalis/view/jobs/train_ai_model.tsx

```diff
@@ -240,3 +393,98 @@ function areBoundingBoxesValid(userBoundingBoxes: UserBoundingBox[]): {
   }
   return { valid: false, reason: "All bounding boxes must have the same size." };
 }
+
+function AnnotationsCsvInput({
+  onAdd,
+}: {
+  onAdd: (newItems: Array<AnnotationWithDataset<APIAnnotation>>) => void;
+}) {
+  const [value, setValue] = useState("");
+  const onClickAdd = async () => {
+    const newItems = [];
+
+    const lines = value.split("\n");
+    for (const line of lines) {
+      const [annotationUrlOrId] = line.trim().split(",");
```

Comment:
Isn't there a validation rule disallowing `,` in each line? Also see my feedback from testing regarding that.